3_Automation/06_Bestiary.yml

```diff
@@ -10,110 +10,63 @@ Name:                                 # must be unique
     INT: ±X
     STR: ±X
     VIT: ±X
-  Passives:
-    Passive: Descrip                  # Any text here
-  Vulnys:
-    Vulny: Descrip
-  Actions:
-    Name:                             # Note: for Companions, type is action for Dealer
-      Type: Major, Minor, Adversary, Free, House # Required.
-      Trigger: Descrip                # Condition of Free Actions
-      To hit: ±X                      # + is implied if not explicit
-      Damage: X
-      Damage type:                    # e.g., if poison, implies bypass AP
-      Range: X                        # 0 is default for 'in melee range'
-      AOE: (aura, cone, line) X       # line would be X by Y, requires new diagram
-      Target: X                       # number: 1 for single, 2 for 2 targets
-      Effect: Descrip
-      Save:
-        Trigger: Once, On hit, Start turn, etc
-        DR: X
-        Type: MOD                     # AGL, CON, etc
-        Fail: Descrip                 # effect on fail
-        Succeed: Descrip              # effect on success. Assume no access to above
   Threat Level: X                     # Consistent comparable rating fitting formula
```

Comment:
This is the only thing I would remove before merging as we don't follow-through yet. 

Otherwise, ya we can add full action specifications later

Reply:
Removed TL from current Bestiary